packages/cashier/src/Components/Error/cashier-locked.jsx

```diff
@@ -91,6 +92,15 @@ const CashierLocked = ({
                     ]}
                 />
             );
+        } else if (is_identity_verification_needed) {
+            message = (
+                <Localize
+                    i18n_default_text='Please submit your <0>proof of identity</0> to authenticate your account and access your Cashier.'
+                    components={[
+                        <a key={0} className='link' rel='noopener noreferrer' href={'/account/proof-of-identity'} />,
```

Comment:
```suggestion
                        <a key={0} className='link' href={'/account/proof-of-identity'} />,
```
This is not needed for internal links

Reply:
Done. Thanks for the suggestion @yashim-deriv